packages/client/layouts/image-custom.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
```

Comment:
I am not sure about this layout. I am concerned about the slope effect that we will end up with all CSS rules. I'd recommend creating your own local layouts for your specific needs.

Reply:
Sorry for accidentally deleting the branch, fix that & I have [opened new PR](https://github.com/slidevjs/slidev/pull/862) so we can continue from there.